parachain/primitives/core/src/outbound.rs

```diff
@@ -15,6 +15,9 @@ pub trait OutboundQueue {
 
 	/// Submit the message ticket for eventual delivery to Ethereum
 	fn submit(ticket: Self::Ticket) -> Result<MessageHash, SubmitError>;
```

Comment:
I was thinking of rather having a single submit method in the trait, which takes a priority parameter:

```suggestion
	fn submit(ticket: Self::Ticket, priority: Priority) -> Result<MessageHash, SubmitError>;
```

The underlying implementation of the trait can still have two different submit method though.

Reply:
https://github.com/Snowfork/snowbridge/pull/949/commits/ac791fd9855ff1c70e1c5e2861bff9f8977d8057